README.md

```diff
@@ -424,10 +450,10 @@ instance in the `us-west` region. The `model-training` step then runs on the
 newly-launched instance.
 
 > :tada: **Note that you can use any container with this workflow!** While you
-> must [have CML and its dependencies set up](#install-cml-as-a-package) to use
-> functions such `cml-send-comment` from your instance, you can create your
-> favourite training environment in the cloud by pulling the Docker container of
-> your choice.
+> must [have CML and its dependencies set up](#local-package) to use functions
+> such `cml-send-comment` from your instance, you can create your favourite
+> training environment in the cloud by pulling the Docker container of your
+> choice.
 
 We like the CML container (`docker://dvcorg/cml`) because it comes loaded with
```

Comment:
Independently of whether _we like_ our own container images or not, I'd propose to keep a separate section with the only purpose of documenting our container images and tags.

Having to explain several times that container image tag documentation is buried at the bottom of a section named _«Allocating Cloud Compute Resources with CML»_ does not seem especially appealing. 😈 

```suggestion
#### Container images
We like the CML container (`docker://dvcorg/cml`) because it comes loaded with
```